src/status_im/contexts/communities/actions/airdrop_addresses/view.cljs

```diff
@@ -8,26 +8,23 @@
     [utils.re-frame :as rf]))
 
 (defn- render-item
-  [item]
+  [item _ _ airdrop-address]
   [quo/account-item
    {:account-props item
+    :state         (if (= airdrop-address (:address item)) :selected nil)
+    :on-press      (fn []
+                     (rf/dispatch [:communities/set-airdrop-address (:address item)])
+                     (rf/dispatch [:navigate-back]))
     :emoji         (:emoji item)}])
 
-(defn- accounts-list
-  [{:keys [accounts]}]
-  [rn/view {:style style/account-list-container}
-   (when (seq accounts)
-     [rn/flat-list
-      {:data      accounts
-       :render-fn render-item
-       :key-fn    :address}])])
-
 (defn view
   []
   (let [{id :community-id}          (rf/sub [:get-screen-params])
         {:keys [name images color]} (rf/sub [:communities/community id])
-        logo-uri                    (get-in images [:thumbnail :uri])
-        accounts                    (rf/sub [:wallet/accounts-with-customization-color])]
+        accounts                    (rf/sub [:wallet/accounts-with-customization-color])
+        selected-addresses          (rf/sub [:communities/selected-permission-addresses])
+        selected-accounts           (filter #(contains? selected-addresses (:address %)) accounts)
```

Comment:
The binding `selected-accounts` is computed solely from the result of two subs `:communities/selected-permission-addresses` and `:wallet/accounts-with-customization-color` that don't require any argument (data fully stored in the app-db).

With that in mind, we should have a separate layer-3 subscription and unit test it. This is kind of a general strategy in re-frame to keep views dumb.